### ISSUE TITLE
Sometimes messages are delivered with delay of up to 10 minutes on status.prod fleet

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#14647]
Currently, there are no specific steps of how to reproduce the bug.  Will keep on investigation and update this issue when more details will be found out. 

Sometimes messages stop being delivered on status.prod fleet (Relogin/clearing chat history doesn't help). After some period of time (up to 10 minutes) the sent messages are delivered to receiver. 

https://user-images.githubusercontent.com/97245802/209679417-dbea3c6f-0fa3-4edf-ad3a-54079abc2423.mp4

[sender_logs.log](https://github.com/status-im/status-mobile/files/10308647/sender_logs.log)

Attaching sender device logs, maybe they might be helpful in finding the root cause. 

[sender_geth.log](https://github.com/status-im/status-mobile/files/10308659/sender_geth.log)
[sender_status.log](https://github.com/status-im/status-mobile/files/10308660/sender_status.log)

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by pavloburykh** [comment]
### UPDATE

We had a call session with @qfrank. During the call we tested interaction between our users, e.g. sending contact requests, creating group chats, adding each other as members of group, messaging etc. 

We figured out that most of the time @qfrank had experienced following issues:

1. Contact requests that I've been sending to him appeared only after app relaunch. Before relaunch AC remained empty. 
2. Adding @qfrank as a member of a newly created group did not result in group to appear in his list on Messages screen.

Though, sometimes everything was okay and requests were delivered once they have been sent.

We had an assumption that it could be the result of great fire wall in Chine (@qfrank is currently located there) which might block some traffic and resources. So we tried to test Status app release version 1.20.2 and it worked fine, without any issues with delivery of contacts requests, groups, messages etc. 

It worth mentioning that @qfrank faces same issues on Status desktop. 

Below I attached logs from @qfrank iPhone and from my device, maybe it might help somehow. 

pavlo_logs.zip - are mine logs as contact request sender
qfrank_logs.zip - are @qfrank logs as contacts request receiver

[pavlo_logs.zip](https://github.com/status-im/status-mobile/files/10353218/pavlo_logs.zip)
[qfarnk_logs.zip](https://github.com/status-im/status-mobile/files/10353219/qfarnk_logs.zip)

**Comment by pavloburykh** [comment]
Recent example of messages delivery delay in community channel. Captured on PR build https://github.com/status-im/status-mobile/pull/14700

https://user-images.githubusercontent.com/97245802/212297179-02a858cf-f8de-4206-87dc-ca699be97820.mp4

[receiver_logs.zip](https://github.com/status-im/status-mobile/files/10410565/receiver_logs.zip)
[sender_logs.zip](https://github.com/status-im/status-mobile/files/10410566/sender_logs.zip)

Also, we are often facing issue of delay in delivery of contact requests. User A adds User B to contact list. Actual result: User B receives User's A CR after couple of minutes.

**Comment by pavloburykh** [comment]
Updated cases in new issue https://github.com/status-im/status-mobile/issues/14777 Closing this one as duplicate.
